### DISCUSSION TITLE
Kelime sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#289] [Zeka Sorusu]
ŞAB KŞAPA olursa LE ne olur?

**Comment by AhmetEmirErsoz** [comment]
ŞAB dan BAŞ kelimesi türer KŞAPA dan da ŞAPKA 
LE, EL olduğunu için LE, ELDİVEN olur

**Reply by sonsuzus** [reply]
Eldiven şeklinde mi yazılır?

**Reply by HabilCcabbar** [reply]
İELDVEN?

**Reply by HabilCcabbar** [reply]
EELDİVN?
